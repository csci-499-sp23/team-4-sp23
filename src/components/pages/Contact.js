import React from 'react';
const Contact = () => {
    return (
        <div className="container p-3 text-center">
            <h2>Contact Us</h2>

            <h3>Send us your email and we will get back to you!</h3>

            <form >
                <div className="mb-3">
                    <label for="exampleFormControlInput1" className="form-label" action='/contact.php' method="get">Email address</label>
                    <input type="email" className="form-control" id="exampleFormControlInput1" name="email"></input>
                </div>
                <div className="mb-3">
                    <label for="exampleFormControlTextarea1" className="form-label">Issue</label>
                    <textarea className="form-control" id="exampleFormControlTextarea1" rows="5" name="issue"></textarea>
                </div>
                <button type="submit" className="btn btn-primary">Submit</button>
            </form>

        </div>
    );
};
export default Contact;
