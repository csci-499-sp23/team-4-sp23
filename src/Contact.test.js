import renderer from 'react-test-renderer';
import Contact from './components/pages/Contact.js';

it('renders correctly',() => {
    const tree = renderer
    .create(<Contact page = 'https://wheel-call-you-9b6ca.firebaseapp.com/contact'>Contact Us</Contact>)
    .toJSON();
    expect(tree).toMatchSnapshot();
})