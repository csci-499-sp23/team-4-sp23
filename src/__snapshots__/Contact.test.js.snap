// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container-fluid"
>
  <h2>
    Contact Us
  </h2>
  <h3>
    Send us your email and we will get back to you!
  </h3>
  <form>
    <div
      className="mb-3 fields"
    >
      <label
        action="/contact.php"
        className="form-label"
        for="exampleFormControlInput1"
        method="get"
      >
        Email address
      </label>
      <input
        className="form-control"
        id="exampleFormControlInput1"
        name="email"
        type="email"
      />
    </div>
    <div
      className="mb-3 fields"
    >
      <label
        className="form-label"
        for="exampleFormControlTextarea1"
      >
        Issue
      </label>
      <textarea
        className="form-control"
        id="exampleFormControlTextarea1"
        name="issue"
        rows="5"
      />
    </div>
    <button
      className="btn btn-primary"
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
